charts/kyverno/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: kyverno
-version: v2.0
+version: v2.1
```

Comment:
Can we update the version to `v2.0.1`, it would be best to follow the same versioning strategy with Kyverno (major.minor.patch).

Reply:
Absolutely, I wasn't sure when I bumped it. The fix is coming asap.

Reply:
fixed 👍